docs/release-notes/NuGet-6.2.md

```diff
@@ -0,0 +1,96 @@
+---
+title: NuGet 6.2 Release Notes
+description: Release notes for NuGet 6.2 including new features, bug fixes, and DCRs.
+author: martinrrm
+ms.author: mruizmares
+ms.date: 5/9/2022
+ms.topic: conceptual
+---
+
+# NuGet 6.2 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.2.0**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.2](https://visualstudio.microsoft.com/downloads/) | [6.0.202](https://dotnet.microsoft.com/download/dotnet-core/6.0)<sup>1</sup> |
```

Comment:
Not sure if [6.0.202] is the correct version, I just grabbed the latest one

Reply:
Accroding to https://github.com/NuGet/Client.Engineering/issues/1560 ; NuGet 6.2 ships in dotnet SDK 6.0.3xx and later

Reply:
It'll be 6.0.300 actually.

Reply:
Changed to 6.0.300